suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -101,7 +101,7 @@ export const groupTransactionsByDate = (
                 const key =
                     !isTxPending && item.blockTime && item.blockTime > 0
                         ? keyFormatter(new Date(item.blockTime * 1000))
-                        : 'pending';
+                        : `pending-${keyFormatter(new Date(item.blockTime ? item.blockTime * 1000 : new Date()))}`;
```

Comment:
not a strong opinion

How about changing this method that it will return 

```
{
    [key: string]: {pending: WalletAccountTransaction[]; success: WalletAccountTransaction[];}
}

or 

{
  pending: {
      [key: string]: WalletAccountTransaction[];
  };
  success: {
      [key: string]: WalletAccountTransaction[];
  };
}
```

instead of 
```
{
    [key: string]: WalletAccountTransaction[];
}
```
with the "hacky" pending date